### ISSUE TITLE
Broken Parent-Child Behavior in Accessibility Tree on `Tabs Bar`

### ISSUE THREAD

**Issue by o-hook** [porsche-design-system/porsche-design-system#2184]
Should be merged with https://github.com/porsche-design-system/porsche-design-system/issues/2193

## Description

In the `Tabs Bar` component, the parent-child nesting of `role="tablist"` and `role="tab"` is broken. Due to nesting slotted content inside slotted content, additional wrapping div's are created which break the rule of `tablist` must have `tab` element as direct children. 

Due to the reason that this issue occurs in many product teams, we should fix it in PDS `v2` and `v3`

## To Reproduce

1. Go to https://designsystem.porsche.com/v2/components/tabs-bar/examples 
2. Inspect `Tabs Bar` example with DevTools
3. Select nested slot inside `scroll-wrapper`
4. Switch into Accessibility-Tree View
5. See false semantic nesting of `tab` elements

## Corresponding Slack Chat

- https://porschedev.slack.com/archives/CT7AVHTTQ/p1669108502138769 
- https://porschedev.slack.com/archives/CT7AVHTTQ/p1674664465765599

## Expected behavior

`tab` elements should semantically placed as direct children of `tablist`

## Screenshots

<img width="653" alt="Bildschirm­foto 2022-11-22 um 11 12 11" src="https://user-images.githubusercontent.com/40788023/203290543-6ce7e7ab-ed50-4f54-98ff-03a599292db4.png">

## Assumptions how to fix

Add `role="presenation"` to all wrapping `div` elements to ignore them from the accessibility tree.

## Other to do's
- If fix is implemented, enable the rule on PDS Storefront.

## Test Environment

Platform/Browser/Version: all

**Comment by o-hook** [comment]
After research I came to the following conclusion:
- The issues found by axe-core rely not on the generic elements because they are ignored
- The issues found by axe-core rely on other child elements which have another role than `listitem` e.g. `button` 
- In case of the `button` issue it should not be marked as an issue by axe-core because they have `aria-hidden="true"` applied and should be ignored. This bug is mention here: https://github.com/dequelabs/axe-core/issues/3850
- In case of the other element with `tabindex="0"` we can try to add an additional `role="none"` and wait until axe-core 2.7 is released and test if this issue still occurs.
